### DISCUSSION TITLE
R9 SX

### DISCUSSION THREAD

**Discussion by CForst1965** [ExpressLRS/ExpressLRS#3000] [General]
Hello has anyone flashed the frsky R9 SX receiver?
I was trying to track down a schematic. They've removed the chip ID but I'd say it's similar to the mini with diversity.  Colin.

**Comment by pkendall64** [comment]
No, it's not supported. Also, the ExpressLRS dev team are removing STM32 support in the next major release.

**Reply by CForst1965** [reply]
Thanks, I'll stick to getting esp elrs.
